app/templates/share/files.html

```diff
@@ -18,4 +18,61 @@
   </div>
 </div>
 
```

Comment:
Same comment here as in the share/views.py comment. This upload form will be used for uploading any file, not just PRMS files. I can change the language here to instruct the user what to do. Please change the form to reflect the fact that any file or a set of files might be uploaded through this interface.